communication/vehicleconnections/vehicleconnection.h

```diff
@@ -43,6 +43,10 @@ class VehicleConnection : public QObject
         NoSystem,
         ParamValueTooLong,
     };
+    enum class Gearbox {
+        lowGear = 1000,
+        highGear = 2000,
+    };
```

Comment:
For supporting more general cases in the future, would it not make more sense to have a mapping from gear number to pwm? Probably an array would to the job. But this mapping should be on the vehicle side. It is vehicle specific, not vehicleConnection specific.